packages/smooth_app/lib/pages/scan/ml_kit_scan_page.dart

```diff
@@ -190,7 +190,7 @@ class MLKitScannerPageState extends State<_MLKitScannerPageContent> {
     }
 
     stoppingCamera = true;
-    if (mounted) {
+    if (!fromDispose && mounted) {
```

Comment:
I would feel safer the other way around: `mounted && !fromDispose` (I never know if `!` is stronger than `&&`).

Reply:
I will do it otherwise, to have a "true" variable for the first part